pywbem/_logging.py

```diff
@@ -14,404 +14,341 @@
+    Thus, for example, the following could be used to configure and
+    activate both loggers::
+
+        configure_loggers_from_string('api=file:summary,http=stderr:all',
+                                      log_filename='mylog.log',
+                                      connection='True')
+
+.. _`Logging configuration Examples`:
+
+Logging configuration examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Configure and activate loggers with
+:meth:`~pywbem.WBEMConnection.configure_logger`.
+
+* Configure all WBEMConnections api logger for summary information output to
+  a file::
+
+    WBEMConnection.configure_logger('api', dest='stderr',
+                                    detail_level='summary',
+                                    connection=True)
+    conn = WBEMConnection()  # logs WBEMConnection api calls and responses
+
+* Configure a single WBEMConnection for output of summary information::
+
+    conn = WBEMConnection()
+    WBEMConnection.configure_logger('all', dest='file', log_filname='xxx.log',
+                                        connection=conn)
+
+* Configure logging to log only http requests and responses including all
+  information in the requests and responses to a file::
+
+    WBEMConnection.configure_logger('http', 'file', log_filename='mylog.log')
+    conn = WBEMConnection(..., enable_log='all')
+
+Configure pywbem logging using Python logging methods.
+
+* Configure a single pywbem connection with standard Python logger methods by
+  defining the root logger with basicConfig::
+
+    # Set basic logging configuration (i.e root level logger)
+    import logging
+    logging.basicConfig(filename='example.log',level=logging.DEBUG)
+    conn = WBEMConnection(...)
+
+    # Define the detail_level and WBEMConnection object to activate.
+    WBEMConnection.configure_logger('all', detail_level='all', connection=conn)
+
+
+* Configure the api logger  with a RotatingFileHandler for all subsequent
+  WBEMConnections using Python methods to configure the loggers so all `pywbem`
+  loggers are configured::
+
+    from logging.handlers import RotatingFileHandler
+    # Define logger for pywbem name
+    logger = logging.getLogger('pywbem')  # logger for pywbem.http and pywbem.api
+    handler = logging.RotatingFileHandler("my_log.log", maxBytes=2000,
+                                          backupCount=10)
+    handler.setLevel(logging.DEBUG)
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(message)s')
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+
+    # configure detail_level and set connection so all WBEMConnection objects
+    # are activated.
+    WBEMConnection.configure_logger('api', detail_level='summary',
+                                    connection=True)
+
+    # All of the following connections do api call and response logging with
+    # summary information.
+    WBEMConnection.WBEMConnection(...)
+    WBEMConnection.WBEMConnection(...)
```

Comment:
The above should probably be something like:
```
conn1 = WBEMConnection(...)
conn2 = WBEMConnection(...)
```

Reply:
DONE